embulk-jruby-extension/src/main/java/org/jruby/util/Temporal.java

```diff
@@ -0,0 +1,90 @@
+package org.jruby.util;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+public class Temporal // TODO better naming
+{
+    // @see https://github.com/jruby/jruby/blob/master/core/src/main/java/org/jruby/RubyTime.java#L1366
```

Comment:
This link comment to the master branch with line number doesn't make sense because the line changes when someone changes the code.